src/net_processing.cpp

```diff
@@ -3974,7 +3978,7 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
 
         // Assume that this is in response to any outstanding getheaders
         // request we may have sent, and clear out the time of our last request
-        peer->m_last_getheaders_timestamp.store(NodeSeconds{});
+        WITH_LOCK(peer->m_last_getheaders_mutex, peer->m_last_getheaders_timestamp = {});
```

Comment:
`NodeSeconds{}`, as well as `std::chrono::time_point<NodeClock>{}` will construct a time point representing the clock's epoch, see https://en.cppreference.com/w/cpp/chrono/time_point/time_point. The duration (/time) since epoch is always zero, regardless of the durations period. So this should be fine as is.

Reply:
Alright, zero is always zero even if the period is `std::ratio<1>` for one and something else for the other.

This means that it should be possible to use just `{}` in the declaration for consistency, right?

```diff
-NodeClock::time_point m_last_getheaders_timestamp GUARDED_BY(m_last_getheaders_mutex) {NodeSeconds{}};
+NodeClock::time_point m_last_getheaders_timestamp GUARDED_BY(m_last_getheaders_mutex) {};
```